lib/dor/services/client/object_version.rb

```diff
@@ -69,7 +70,10 @@ def open(**params)
         end
 
         # Close current version for an object
-        # @param params [Hash] optional params (see dor-services-app)
+        # @param :description [String] (optional) - a description of the object version being opened
```

Comment:
There shouldn't be any colons on these lines. See https://www.rubydoc.info/gems/yard/file/docs/Tags.md#param

Reply:
whoops - that one slipped through.